webhooks-extension/docs/Limitations.md

```diff
@@ -0,0 +1,57 @@
+# Limitations
+<br/>
+
+- Only GitHub webhooks are currently supported.
+- Only `push` and `pull_request` events are currently supported, these are the events defined on the webhook.
+- A limited number of pipelines will currently work with this implementation (see below)
+
+## Pipeline Limitations
+
+It is worth noting that the webhooks extension does not currently work with all pipelines, it very specifically creates the following when the webhook is triggered:
+
+#### Git PipelineResource
+
+A PipelineResource of type git is created with:
```

Comment:
of type `git` (markdown) as `git` is a reserved term

Reply:
done